code/game/machinery/kitchen/monkeyrecycler.dm

```diff
@@ -114,25 +114,11 @@
 	if(.)
 		return
 	if(grinded >= required_grind)
-		if(connected)
-			var/whereto = alert(user, "Where to send recycled monkeys?", "Xenobiology", "Outside", "In management console")
-			switch(whereto)
-				if("Outside")
-					to_chat(user, "<span class='notice'>The machine hisses loudly as it condenses the grinded monkey meat. After a moment, it dispenses a brand new monkey cube.</span>")
-					for(var/i = 0, i < cube_production, i++)
-						new /obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped(loc)
-				if("In management console")
-					connected.monkeys += cube_production
-					to_chat(user, "<span class='notice'>[connected] now has [connected.monkeys] monkeys stored.</span>")
-				else
-					return
-		else
-			to_chat(user, "<span class='notice'>The machine hisses loudly as it condenses the grinded monkey meat. After a moment, it dispenses a brand new monkey cube.</span>")
-			for(var/i = 0, i < cube_production, i++)
-				new /obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped(loc)
-		grinded -= required_grind
+		to_chat(user, "<span class='notice'>The machine hisses loudly as it condenses the grinded monkey meat. After a moment, it dispenses a brand new monkey cube.</span>")
 		playsound(src, 'sound/machines/hiss.ogg', VOL_EFFECTS_MASTER)
+		grinded -= required_grind
+		for(var/i = 0, i < cube_production, i++)
```

Comment:
https://github.com/TauCetiStation/TauCetiClassic/wiki/Code-Convention#%D0%A0%D0%B0%D0%B1%D0%BE%D1%82%D0%B0-%D1%81-%D1%86%D0%B8%D0%BA%D0%BB%D0%B0%D0%BC%D0%B8-black-magic